packages/react-component-library/src/components/TabSet/TabSet.tsx

```diff
@@ -163,12 +170,7 @@ export const TabSet: React.FC<TabSetProps | ScrollableTabSetProps> = ({
         {Children.map(
           children,
           (child: React.ReactElement<TabProps>, index: number) => {
-            const {
-              children: tabChildren,
-              title,
-              isActive,
-              ...tabRest
-            } = child.props
+            const { children: tabChildren, title, ...tabRest } = child.props
```

Comment:
Think we need to destructure `isActive` to drop it from `tabRest` as its set below.

Reply:
I rebased so this commit should be updated.